planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/utils/calculation.cpp

```diff
@@ -382,30 +382,58 @@ std::vector<double> calc_lon_acceleration_samples(
 }
 
 double calc_lane_changing_acceleration(
-  const double initial_lane_changing_velocity, const double max_path_velocity,
-  const double lane_changing_time, const double prepare_longitudinal_acc)
+  const CommonDataPtr & common_data_ptr, const double initial_lane_changing_velocity,
+  const double max_path_velocity, const double lane_changing_time,
+  const double prepare_longitudinal_acc)
 {
   if (prepare_longitudinal_acc <= 0.0) {
-    return 0.0;
+    const auto & params = common_data_ptr->lc_param_ptr->trajectory;
+    const auto lane_changing_acc = common_data_ptr->transient_data.is_ego_near_current_terminal_start
+                                    ? prepare_longitudinal_acc * params.lane_changing_decel_factor
+                                    : 0.0;
+    return lane_changing_acc;
```

Comment:
I think it might be better to either 
1. separate this into another PR, and merge this PR together to ensure no degradation, or 
2. renaming this PR to something other. 

I prefer the first, as it is more proper. I understand that the fix is introduced due to scenario degradation, but the reason for adding acceleration related fix into blinker's PR seems a little vague.

It also save us a little trouble when cherry-picking things.

Reply:
Yes sir :saluting_face:

Reply:
@zulfaqar-azmi-t4 
https://github.com/autowarefoundation/autoware.universe/pull/9545